### PR TITLE
MCR-2998 add save-xsl to xslt folder

### PR DESCRIPTION
**Pull request by michael-becker**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2998)

### CODE REVIEW

File: mycore-base/src/main/java/org/mycore/frontend/cli/MCRObjectCommands.java [hunk 1]
```diff
@@ -579,7 +579,8 @@ public static void showLastID(String base) {
     @MCRCommand(
         syntax = EXPORT_OBJECT_TO_DIRECTORY_WITH_STYLESHEET_COMMAND,
         help = "Stores the object with the MCRObjectID {0} to the directory {1}" +
-            " with the stylesheet {2}-object.xsl. For {2}, the default is xsl/save.",
+            " with the stylesheet {2}-object.xsl. For {2}, the default is "+
```
Comment (rsteph-de) [inline mycore-base/src/main/java/org/mycore/frontend/cli/MCRObjectCommands.java]: format code - space before '+')?

**Review by rsteph-de — CHANGES_REQUESTED** [review]
We need to modify the other export command to.
Please look into MCRDerivateCommands and MCRClassification2Commands

**Comment by rsteph-de** [comment]
I would like do to some additional cleanup in the MCRObjectCommands class.
You could simply remove the class from this Pull-Request and I will update the help text later

File: mycore-base/src/main/java/org/mycore/frontend/cli/MCRObjectCommands.java (hunk 1 above)
Comment (michael-becker) [reply mycore-base/src/main/java/org/mycore/frontend/cli/MCRObjectCommands.java]: I removed the file from the commit

**Comment by michael-becker** [comment]
> I would like do to some additional cleanup in the MCRObjectCommands class. You could simply remove the class from this Pull-Request and I will update the help text later

I removed the file and now only the xsls are in the commit

**Review by rsteph-de — APPROVED** [review]
O.K.

**Pull request merged**